src/components/ShareSocialCta/index.js

```diff
@@ -30,7 +30,7 @@ function ShareSocialCta({
 }) {
   return (
     <section className={styles.container}>
-      <a name="share" />
+      <a id="`${{location.pathname}}`#share" aria-label={"Share"}>{null}</a>
```

Comment:
this is great work 👍 however, this can just be id='share', rather than the location.pathname, as whichever href is pointing to it will know to add it to the end of the current url.

You can see that when you click the small share icon in the corner, it jumps to the top as it doesn't know where this share anchor is.

Reply:
Sure